backend/data/static/create_master_files.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import os
+from functools import reduce
+
+# Set up directory for the os
+directory = "cleaned/"
+
+# Initialise some values for the loop
+la = {}
+i = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
```

Comment:
another potential issue:

`os.listdir` returns the list of **both** files AND directories. If it happens that a directory is there, also matching that name pattern, this code will break.

Two options:
- checking if the path (`os.path.join(DIRECTORY, filename)` is file - `os.path.isfile`;
- using `os.walks` - a bit more complicated to use apparently but that returns `directories` and `filename` as two separated lists so no risk to mix:

```
root, dirnames, filenames = next(os.wa;lk(DIRECTORY))  
# using walk cause we just need one level, no need to go further in sub dirs
for filename in enumerate(filenames:
    if filename.startswith('LA_'):
       filepath = os.path.join(root, filename)
       la[...] = pd.read_csv(filepath, ...)
```

Reply:
RE "no risk to mix" - also, no need to check if the file exists nor if that is a directory either ;)